cpp/README.md

```diff
@@ -0,0 +1,26 @@
+# IronOxide-Cpp
+
+The building of the C++ library is currently not done by CI, so if you want to produce C/C++ bindings and the associated
```

Comment:
Except for the iOS library; we're doing that in CI. Is it worth mentioning that here? It'd probably just confuse things, right?

Reply:
Actually, we do perform the C++ build in CI to make sure it's all working. It's not build by CD and published in binary form. If we say we're not even doing CI builds, that's sort of a red flag that we don't care.

Reply:
This file is just a rename of Colt's original `BUILDING.md`, so it is most likely out of date. 
I'll just remove this section about CI.